docs/undocumented.json

```diff
@@ -1,978 +1,1020 @@
 {
   "warnings": [
     {
-      "file": "/Users/raulriera/Downloads/Shopify/FunctionalTableData/FunctionalTableData/CellActions.swift",
+      "file": "/Users/scottcampbell/src/github.com/Shopify/FunctionalTableData/FunctionalTableData/CellActions.swift",
```

Comment:
Maybe this file shouldn't have been committed 🤔 but that was on me

Reply:
Looks like I can just remove the `undocumented.json` 🤷‍♂️

Reply:
Kill it, and maybe gitignore? 🤔

Reply:
👍